functions/Restore-DbaBackup.ps1

```diff
@@ -0,0 +1,215 @@
+function Restore-DbaBackup
+{
+<#
+.SYNOPSIS 
+Restores a SQL Server Database from a set of backupfiles
+
+.DESCRIPTION
+Scans a given folder for Full, Differential and Log backups. 
+OR
+Takes a set of folder arrays and processes them for Full, Differential and Log backups.
```

Comment:
Works it way through all the files, works out which ones are legitimate backup files. Works out which ones of those are required to perform the request restore (latest or point in time), and check we have an unbroken restore chain to do that with,

Would that work as a description?